app/app.js

```diff
@@ -103,7 +103,7 @@ module.exports = (options) => {
     const globalData = (type === 'layout' && exampleConfig.data) || {};
 
     if (!exampleConfig) {
-      next();
+      return next();
```

Comment:
for a component that had examples if you requested an example which didn't exist you'd get a misdirecting error logged rather than just not finding the example